### ISSUE TITLE
training cnn from scratch with the number of channels in input is greater than 3

### ISSUE THREAD

**Issue by pxnguyen** [vlfeat/matconvnet#556]
Hi,

I am struggling to training a vgg-vd, vgg-m or even vgg-s using flows data (input is 224x224x10 instead of  RGB image, 224x224x3). It seems like the CNN model is greatly under-fitting the data. Top-1 error is at the random chance error. I have tried out different architecture/learning rate/batch size/normalization. Still can't make the CNN learn something better than random chance.

Just wondering if anyone has successfully implemented something similar?

More details:
I tried to train both from scratch or initialized from the imagenet-vgg16. Both failed terribly.
Attached are the examples of flow input images:
![flow_x_0007](https://cloud.githubusercontent.com/assets/1129146/15539720/f1fc6f46-2238-11e6-84a6-ffad38326fad.jpg)
![flow_x_0024](https://cloud.githubusercontent.com/assets/1129146/15539838/9bd483e6-2239-11e6-82ef-678ddfe7e312.jpg)
